### ISSUE TITLE
Unable to install chrome driver version 2.20

### ISSUE THREAD

**Issue by tnguyen14** [vvo/selenium-standalone#142]
When I try to install chromedriver version 2.20, it falls back to version 2.2 instead

```
18:00 🚀  ~
:; selenium-standalone install --drivers.chrome.version=2.20 --drivers.chrome.baseURL=http://chromedriver.storage.googleapis.com
----------
selenium-standalone installation starting
----------

---
selenium install:
from: http://selenium-release.storage.googleapis.com/2.47/selenium-server-standalone-2.47.1.jar
to: /Users/tnguyen/.dotfiles/node_modules/selenium-standalone/.selenium/selenium-server/2.47.1-server.jar
---
chrome install:
from: http://chromedriver.storage.googleapis.com/2.2/chromedriver_mac32.zip
to: /Users/tnguyen/.dotfiles/node_modules/selenium-standalone/.selenium/chromedriver/2.2-x64-chromedriver


selenium-standalone installation [====================] 100% 0.0s

-----
selenium-standalone installation finished
-----
```

and
```
18:00 🚀  ~
:; ll ~/.dotfiles/node_modules/selenium-standalone/.selenium/chromedriver/
total 49584
drwxr-xr-x  4 tnguyen  staff       136 Oct 21 18:00 ./
drwxr-xr-x  4 tnguyen  staff       136 Oct 21 18:00 ../
-rwxr-xr-x  1 tnguyen  staff  17750460 Oct 21 18:00 2.2-x64-chromedriver*
-rw-r--r--  1 tnguyen  staff   7634565 Oct 21 18:00 2.2-x64-chromedriver.zip
```

**Comment by vvo** [comment]
Interesting, could you help debug this? git clone, npm run link, do the install command + console log the source

**Comment by tnguyen14** [comment]
Hi,

I have tried to do so according to your instructions. Below is the result (which is the same as before). I'm guessing what you meant by console log the source is running `ll` on the chromedriver folder? If there's something else you need me to do, please let me know.

```
22:47 🚀  ~/temp/test
:; selenium-standalone install --drivers.chrome.version=2.20 --drivers.chrome.baseURL=http://chromedriver.storage.googleapis.com
----------
selenium-standalone installation starting
----------

---
selenium install:
from: http://selenium-release.storage.googleapis.com/2.47/selenium-server-standalone-2.47.1.jar
to: /Users/tnguyen/temp/selenium-standalone/.selenium/selenium-server/2.47.1-server.jar
---
chrome install:
from: http://chromedriver.storage.googleapis.com/2.2/chromedriver_mac32.zip
to: /Users/tnguyen/temp/selenium-standalone/.selenium/chromedriver/2.2-x64-chromedriver


selenium-standalone installation [====================] 100% 0.0s

-----
selenium-standalone installation finished
-----
22:47 🚀  ~/temp/test
:; ll ../selenium-standalone/.selenium/chromedriver/
total 49584
drwxr-xr-x  4 tnguyen  staff       136 Oct 21 22:47 ./
drwxr-xr-x  4 tnguyen  staff       136 Oct 21 22:47 ../
-rwxr-xr-x  1 tnguyen  staff  17750460 Oct 21 22:47 2.2-x64-chromedriver*
-rw-r--r--  1 tnguyen  staff   7634565 Oct 21 22:47 2.2-x64-chromedriver.zip
```

**Comment by vvo** [comment]
ok fixed in 4.7.1
